src/test_l1_tvSettings.c

```diff
@@ -12015,10 +12016,10 @@ void test_l1_tvSettings_negative_GetColorTemp_B_post_offset_onSource (void)
     for (unsigned int i = 0; i < tvColorTemp_MAX; i++)
     {
         platformFlag = false;
-        snprintf(keyValue, UT_KVP_MAX_ELEMENT_SIZE, "tvSettings/ColorTemperature/index/%d", i);
-        colorTempValue = (tvColorTemp_t) UT_KVP_PROFILE_GET_UINT32(keyValue);
         for (unsigned int j = 0; j < colorTempCount ; j++ )
         {
+            snprintf(keyValue, UT_KVP_MAX_ELEMENT_SIZE, "tvSettings/ColorTemperature/index/%d", j);
+            colorTempValue = (tvColorTemp_t) UT_KVP_PROFILE_GET_UINT32(keyValue);
             if(colorTempValue == (tvColorTemp_t)i)
```

Comment:
"In line number 12032, are we supposed to pass the index i instead of colorTempValue? This is because we intend to pass the unsupported enum value."

Reply:
updated